docs/docs/testing-your-assistant.mdx

```diff
@@ -72,6 +72,26 @@ Here are some examples:
     - action: utter_ask_price
   ```
 
+  </TabItem>
+  <TabItem value="buttons">
+
+  ```yaml-rasa title="tests/test_stories.yml" {5,12}
```

Comment:
I think here you'd like to highlight lines 8, 11 which contain the user examples for handling buttons.
```suggestion
  ```yaml-rasa title="tests/test_stories.yml" {8,11}
```